documentation/src/main/doc/modules/kafka/pages/protobuf-configuration.adoc

```diff
@@ -0,0 +1,45 @@
+[#kafka-protobuf-configuration]
+== Using Google Protobuf serializer/deserializer
+
+If you are using https://developers.google.com/protocol-buffers/[Protocol Buffers] serializer/deserializer, please note the following configuration properties.
+
+=== For https://docs.confluent.io/platform/current/schema-registry/serdes-develop/serdes-protobuf.html#serdes-and-formatter-protobuf[Confluent] Schema Registry
+
+Confluent protobuf library is `io.confluent:kafka-protobuf-serializer`.
+Note that this library is not available in Maven Central, you need to use the https://docs.confluent.io/clients-kafka-java/current/overview.html[Confluent Maven repository].
+
```

Comment:
Here we may add copy-pasteable maven instructions for adding repository+dependency. I know instructions for Gradle will be missing but that's a separate global effort to improve documentation.